src/Opdex.Platform.Infrastructure/Data/DbContext.cs

```diff
@@ -30,7 +30,7 @@ public async Task<IEnumerable<TEntity>> ExecuteQueryAsync<TEntity>(DatabaseQuery
 
     public async Task<IEnumerable<TReturn>> ExecuteQueryAsync<TFirst, TSecond, TReturn>(DatabaseQuery query,
                                                                                         Func<TFirst, TSecond, TReturn> map,
-                                                                                        string splitOn)
+                                                                                        string splitOn = "Id")
```

Comment:
Another nit, just because we have this done different elsewhere and this can be an easy area to miss causing a bugs in my past experience. 

I think we should remove the default "Id" from here and the interface and instead follow how its done in this query:
https://github.com/Opdex/opdex-v1-api/blob/main/src/Opdex.Platform.Infrastructure/Data/Handlers/Transactions/SelectTransactionsForSnapshotRewindQueryHandler.cs

Reply:
Nice, I've followed how you've done it there, so the mapping is done within the split func rather than the summary being defined on the entity.